asapdiscovery-data/asapdiscovery/data/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Computational chemistry packages developed by the ASAP Discovery Consortium."""
 
-from importlib.metadata import version
-
-__version__ = version("asapdiscovery-data")
+# from importlib.metadata import version
+#
+# __version__ = version("asapdiscovery-data")
```

Comment:
I believe these are important lines for keeping up with versions for the subpackages, any special reason why we are commenting these?

Reply:
reverted in current version!